client/landing/gutenboarding/components/domain-picker/suggestion-item.tsx

```diff
@@ -28,13 +40,47 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 	const domainName = domain.slice( 0, dotPos );
 	const domainTld = domain.slice( dotPos );
 
+	const { domainSearch } = useSelect( select => select( STORE_KEY ).getState() );
+	const fetchAlgo = `/domains/search/${ DOMAIN_SUGGESTION_VENDOR }/${ FLOW_ID }`;
```

Comment:
Just checking: this is enough to differentiate a _gutenboarding_ domain search, right? (As opposed to a standard Calypso signup one).

That is, we won't need to send any additional flow info?

Reply:
From re-reading PCYsg-bor-p2 I think we can pass in another param `ui_algo` to denote where in the flow the search results appear. So something like `/gutenboarding/domain-popover` might be a good identifier?